Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Editor/GraphyEditorStyle.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace Tayx.Graphy
+{
+    internal static class GraphyEditorStyle
+    {
+        public static Texture2D m_logoTexture = null;
+        public static GUISkin m_skin = null;
+        public static GUIStyle m_headerStyle1 = null;
+        public static GUIStyle m_headerStyle2 = null;
+        public static GUIStyle foldoutStyle = null;
+
+        private static string path;
+
+        static GraphyEditorStyle()
```

Comment:
can you add "public"?

Reply:
static constructor can't have access modifier

Reply:
My bad, did not notice it was the constructor!